Fast company

Homegor opened the pull request.

Solexofficial commented on module_1/Week_1/05_lines/lesson_2/index.js:
```diff
@@ -0,0 +1,13 @@
+console.log(myInfoText.length)
+
+// 5) Первый и последний символы строки
+console.log('Первый: ',myInfoText[0], '\nПоследний: ', myInfoText.slice(-1))
```
какой ещё есть способ получить последний символ строки?

Solexofficial commented on module_1/Week_1/05_lines/lesson_3-4/index.js:
```diff
@@ -0,0 +1,12 @@
+
+const userAge = prompt('Сколько вам лет?')
+
+alert(Number(userAge.trim())) // Преобразовано к типу данных number
```
Старайся предугадать поведение. Попробуй в age ввести цифры + символы, что получим? На досуге попробуй обработать это поведение, если непонятно будет пиши в дискорде

Solexofficial commented on module_1/Week_1/05_lines/lesson_6/index.js:
```diff
@@ -0,0 +1,6 @@
+const userText = prompt('Введите текст'.trim());
```
не там прописал trim

Solexofficial replied:
ниже так же

Solexofficial reviewed (approved):
В целом всё хорошо, молодец!✋ аппрув

Homegor committed:
не могу реализовать поиск через map
bookmarks

Homegor committed:
Merge branch 'main' into Fast-Company

Homegor committed:
Готовое задание bookMark

Homegor committed:
Merge remote-tracking branch 'origin/Fast-Company' into Fast-Company

Homegor committed:
rename length to usersCount

Homegor committed:
Много правок

Homegor committed:
handleIncrement & handleDecrement
правки prevState и ограничение счетчика значение которого не может быть ниже нуля

Homegor committed:
Merge pull request #3 from Homegor/tasks

Tasks

Homegor committed:
Merge branch 'main' into Fast-Company

Homegor committed:
handleIncrement & handleDecrement
правки prevState и ограничение счетчика значение которого не может быть ниже нуля

Homegor committed:
Merge remote-tracking branch 'origin/Fast-Company' into Fast-Company
